pkgs/by-name/jo/jocalsend/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchFromGitea,
+  rustPlatform,
+  pkg-config,
+  openssl,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "jocalsend";
+  version = "1.6.1803";
+
+  src = fetchFromGitea {
+    domain = "git.kittencollective.com";
+    owner = "nebkor";
+    repo = "joecalsend";
+    rev = finalAttrs.version;
+    hash = "sha256-nrXUZb4Yi1ttEltzqKUnMLLr5cvhqCxW1iJyo1ErG0w=";
+  };
+
+  cargoHash = "sha256-0yFKJtwQikP6WRDVWgv7b9e0iUS9AdKpx6VTeNAQ4zs=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    openssl
+  ];
+
+  meta = {
+    homepage = "https://git.kittencollective.com/nebkor/joecalsend";
+    description = "Rust terminal client for Localsend";
+    changelog = "https://git.kittencollective.com/nebkor/joecalsend/releases/tag/${finalAttrs.src.rev}";
+    license = with lib.licenses; [ asl20 ];
```

This is not under Apache license, you got confused with [3P-LICENSE.txt](https://git.kittencollective.com/nebkor/joecalsend/src/branch/main/3P-LICENSE.txt)

[LICENSE.md](https://git.kittencollective.com/nebkor/joecalsend/src/branch/main/LICENSE.md)
The License of the project seems unclear to me, while it appears to be a kinda free license, it is not clear or explicit. I think we could call it unfree redistributable. Would be good to ask upstream though...

```suggestion
    license = with lib.licenses; [ unfreeRedistributable ];
```

---

Thank YOU for adding to nixpgks :)

---

Nice! I hadn't even heard of that site until I got an email letting me know jocalsend was featured :)